src/Datastore/EntityMapper.php

```diff
@@ -40,6 +40,23 @@ class EntityMapper
     private $encode;
 
     /**
+     * @var array
+     */
+    private $valueTypes = [
+        "nullValue",
+        "booleanValue",
+        "integerValue",
+        "doubleValue",
+        "timestampValue",
+        "keyValue",
+        "stringValue",
+        "blobValue",
+        "geoPointValue",
+        "entityValue",
+        "arrayValue",
+    ];
```

Comment:
Is there a canonical source of this list? How to make sure it's synced with the API spec?